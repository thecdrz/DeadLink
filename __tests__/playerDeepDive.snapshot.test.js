const { playerDeepDiveEmbed } = require('../lib/embeds');

describe('player deep dive snapshot', () => {
  test('playerDeepDiveEmbed with fields', () => {
    const fields = [
      { name: 'Stats', value: 'Level: 12\n❤️ 92% | ⚔️ 5 kills | ☠️ 0 deaths' },
      { name: 'Session', value: '15m active | 120m traveled' }
    ];
  const embed = playerDeepDiveEmbed({ title: '🎯 Player Deep Dive', fields });
  // Normalize dynamic footer for deterministic snapshots
  if (embed.footer) embed.footer.text = 'Generated on <fixed>';
  expect(embed).toMatchSnapshot();
  });
});
