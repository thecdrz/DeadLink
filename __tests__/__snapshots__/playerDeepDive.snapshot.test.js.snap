// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`player deep dive snapshot playerDeepDiveEmbed with fields 1`] = `
{
  "color": 5793266,
  "fields": [
    {
      "inline": false,
      "name": "Stats",
      "value": "Level: 12
❤️ 92% | ⚔️ 5 kills | ☠️ 0 deaths",
    },
    {
      "inline": false,
      "name": "Session",
      "value": "15m active | 120m traveled",
    },
  ],
  "footer": {
    "text": "Generated on <fixed>",
  },
  "title": "🎯 Player Deep Dive",
}
`;
